backend/btrixcloud/orgs.py

```diff
@@ -870,11 +876,26 @@ async def invite_user_to_org(
         return {"invited": "existing_user"}
 
     @app.post("/orgs/invite-accept/{token}", tags=["invites"])
-    async def accept_invite(token: str, user: User = Depends(user_dep)):
+    async def accept_invite(
+        token: str, rename: OptionalRenameOrg, user: User = Depends(user_dep)
+    ):
         invite = await invites.accept_user_invite(user, token, user_manager)
+        org = await ops.add_user_by_invite(invite, user)
+        if not org:
+            raise HTTPException(detail="invalid_invite", status_code=400)
```

Comment:
Added this check, now detected with additional typing - in theory possible if non-org invite token is used, I think